db/version_set.cc

```diff
@@ -2031,15 +2034,19 @@ void Version::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
   // use autovector in order to avoid unnecessary construction of GetContext
   // objects, which is expensive
   autovector<GetContext, 16> get_ctx;
+
   for (auto iter = range->begin(); iter != range->end(); ++iter) {
+    std::unique_ptr<BlobFileMergeCallback> blob_file_merge_callback(
```

Comment:
Same here. Also, I think one object would be sufficient (which you could create outside the loop).